container/lxd/initialisation.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package lxd
+
+import (
+	"strings"
+
+	"github.com/juju/utils/packaging/config"
+	"github.com/juju/utils/packaging/manager"
+
+	"github.com/juju/juju/container"
+)
+
+var requiredPackages = []string{
+	"lxd",
```

Comment:
lxd-images is in `lxd` proper, so I don't think so. Calling out to that will go away soon as well, and we'll just use the API to import images as soon as the LXD simplestreams code lands.